### ISSUE TITLE
Add split track button to library

### ISSUE THREAD

**Issue by echarlie** [wuvt/trackman#21]
_From @mutantmonkey on December 6, 2016 4:17_

Some DJs have decided that it is okay to log multiple tracks as one and separate them with a slash. This is not the case, so we need to allow existing tracks in the database to be split into multiple ones.

_Copied from original issue: wuvt/wuvt-site#237_

**Comment by echarlie** [comment]
I'm thinking this may be sufficiently infrequent to just accept a mediocre tracklog (they're sequential tracks, usually). Editing tracklogs would partially solve this problem.

**Comment by mutantmonkey** [comment]
Yeah, I just don't want these sorts of things to come up in the search, basically.
